support-frontend/assets/pages/contributions-landing/components/ContributionArticleCount.jsx

```diff
@@ -0,0 +1,453 @@
+    defaultHeaderCopy: string | React$Node,
+}
+
+type ContributionsArticleCountOptOutWithProps = {
+    numArticles: number;
+    isArticleCountOn: boolean;
+    isMobileOnly: boolean;
+    onArticleCountOptOut: () => void;
+    onArticleCountOptIn: () => void;
+    userName: string | null;
+    defaultHeaderCopy: string | React$Node,
+}
+
+// -- Functions -- //
+
+const OPHAN_COMPONENT_ID_ARTICLE_COUNT_OPT_OUT_OPEN = 'contributions-epic-article-count-open';
+const OPHAN_COMPONENT_ID_ARTICLE_COUNT_OPT_OUT_CLOSE = 'contributions-epic-article-count-close';
+const OPHAN_COMPONENT_ID_ARTICLE_COUNT_STAY_IN = 'contributions-epic-article-count-stay-on';
+const OPHAN_COMPONENT_ID_ARTICLE_COUNT_OPT_OUT = 'contributions-epic-article-count-opt-out';
+const OPHAN_COMPONENT_ID_ARTICLE_COUNT_STAY_OUT = 'contributions-epic-article-count-stay-out';
+const OPHAN_COMPONENT_ID_ARTICLE_COUNT_OPT_IN = 'contributions-epic-article-count-opt-in';
+
+
+// -- Components -- //
+export const ArticleCountWithToggle = ({
+  isArticleCountOn,
+  numArticles,
+  onToggleClick,
+  userName,
+  defaultHeaderCopy,
+}: ArticleCountWithToggleProps) => {
+  if (isArticleCountOn && numArticles >= 5) {
+    return (
+      <div css={articleCountHeaderContainerStyles}>
+        <div css={articleCountAboveContainerStyles}>
+          <ArticleCountHeaderCopy numArticles={numArticles} userName={userName} />
+        </div>
+        <div css={articleCountWrapperStyles}>
+          <div css={articleCountTextStyles}>Article count</div>
+          <ButtonLink
+            priority="secondary"
+            onClick={onToggleClick}
+            cssOverrides={articleCountCtaStyles}
+          >
+      on
+          </ButtonLink>
+        </div>
+      </div>
+    );
+  }
+
+  if (!isArticleCountOn) {
+    return (
+      <div css={articleCountHeaderContainerStyles}>
+        <div css={articleCountAboveContainerStyles}>
+          {defaultHeaderCopy}
+        </div>
+        <div css={articleCountWrapperStyles}>
+          <div css={articleCountTextStyles}>Article count</div>
+          <ButtonLink
+            priority="secondary"
+            onClick={onToggleClick}
+            cssOverrides={articleCountCtaStyles}
+          >
+          off
+          </ButtonLink>
+        </div>
+      </div>
+    );
+  }
+
+  return null;
+};
+
+
+export const ContributionsArticleCountWithOptOut = ({
+  numArticles,
+  isArticleCountOn,
+  isMobileOnly,
+  onArticleCountOptOut,
+  onArticleCountOptIn,
+  userName,
+  defaultHeaderCopy,
+}: ContributionsArticleCountOptOutWithProps) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [consentManagementPlatform, setConsentManagementPlatform] = useState(null);
+
+  function openCmp() {
+    if (consentManagementPlatform) {
+      consentManagementPlatform.showPrivacyManager();
+    }
+  }
+
+  useEffect(() => {
+      import('@guardian/consent-management-platform').then(({ cmp }) => {
+        setConsentManagementPlatform(cmp);
+      });
+  }, []);
```

Comment:
This is fancy! Is there a reason we can't use a regular import e.g

```
import { cmp } from '@guardian/consent-management-platform';
```

and then

```
function openCmp() {
      cmp.showPrivacyManager();
}
```

Reply:
Cmp seems to be imported this way elsewhere in the repo e.g. `Footer.jsx` and `analyticsAndConsent.js` but I can't remember why.  @tomrf1 do you by any chance remember from when we paired on this why we decided to import it this way, or is it ok to do a regular import?

Reply:
I'm happy to keep it this way if that's how we use it in other spots!

Reply:
sorry - yes we just copied how it was done elsewhere in the repo, which doesn't necessarily mean it's the correct way!